packages/cli/src/Format.ts

```diff
@@ -54,31 +54,15 @@ Or specify a Prisma schema path
 
     const schemaPath = await getSchemaPathAndPrint(args['--schema'])
 
-    let output: string | undefined
-
-    try {
-      output = await formatSchema({
-        schemaPath,
-      })
-    } catch (err) {
-      if (err.exitCode === 101 || err.stderr?.includes('panicked at')) {
-        throw new RustPanic(
-          /* message */ err.shortMessage,
-          /* rustStack */ err.stack,
-          /* request */ 'format',
-          ErrorArea.FMT_CLI,
-          schemaPath,
-          /* schema */ undefined,
-        )
-      }
-
-      throw err
-    }
+    let output = await formatSchema({ schemaPath })
 
+    // TODO: what's the point of this?
     await getDMMF({
       datamodel: output,
     })
```

Comment:
I think we do this to check that the output is valid

Reply:
That was Tom's guess and mine, too

Reply:
```suggestion
    // Validate the output is valid
    await getDMMF({
      datamodel: output,
    })
```

Reply:
The conversation was resolved but the line didn't change? 🤔

Reply:
I remember pressing `Commit suggestion`, but maybe I have accidentally overridden the commit later. Thanks for noticing